change/@fluentui-react-avatar-32be9545-3830-48b3-bca7-aa6f556ad333.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "none",
+  "comment": "Adding initial implementation of AvatarGroup",
+  "packageName": "@fluentui/react-avatar",
+  "email": "email not defined",
```

Comment:
It looks like something is misconfigured for your yarn change. Could you update this to be your email?